addons/project/static/src/js/tours/project.js

```diff
@@ -107,11 +107,13 @@ tour.register('project_tour', {
     position: "bottom",
     run: "click",
 }, {
-    trigger: ".o_field_widget[name='user_ids'] input",
+    trigger: ".o_field_widget[name='user_ids']",
```

Comment:
I am wondering if it is not equivalent to let the initial trigger and change the run to do a click instead of `text a`? 🤔

Reply:
> I am wondering if it is not equivalent to let the initial trigger and change the run to do a click instead of `text a`? 🤔

According to the bug description, it should function correctly if you simply select the user without needing to input the 'a' manually. 🙂